AzureCommunicationUI/sdk/AzureCommunicationUIChat/Sources/Presentation/SwiftUI/Chat/ChatViewComponents/BottomBarView.swift

```diff
@@ -55,4 +61,9 @@ struct BottomBarView: View {
         IconButton(viewModel: viewModel.sendButtonViewModel)
             .flipsForRightToLeftLayoutDirection(true)
     }
+
+    var localParticipantInfoBanner: some View {
```

Comment:
Would this be a generic banner? or only for participant removed? If so probably use a more descriptive name here

Reply:
only for local participant removed, we don't have any other use case that needs this label

Reply:
Could we rename this to something more descriptive?

Reply:
I thought this is descriptive enough? Do you have any suggestions?

Reply:
Might be more clear if we just add `removed`
```suggestion
    var localParticipantRemovedBanner: some View {
```

Reply:
updated thanks